src/client/pages/OfferNew/Perils/PerilRow.tsx

```diff
@@ -4,15 +4,10 @@ import { InsuranceValues } from 'pages/OfferNew/Perils/InsuranceValues'
 import { PerilCollection } from 'pages/OfferNew/Perils/PerilCollection'
 import { PerilModal } from 'pages/OfferNew/Perils/PerilModal'
 import { OfferQuote } from 'pages/OfferNew/types'
-import { LARGE_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
 
 const InsuranceTypeHeadline = styled.div`
-  padding: 16px 0;
-  font-size: 1.375rem;
-
-  ${LARGE_SCREEN_MEDIA_QUERY} {
-    font-size: 1.5rem;
-  }
+  padding: 1rem 0;
+  font-size: 1rem;
```

Comment:
Should we keep some hierarchy between the title of the section and the title of the perils? It feels a bit small, in desktop, maybe have it as 1.25rem? :)

Reply:
Hm, I think it's nice that it's small. But I actually looked in the **Web UI Kit** file Figma and there's no font size between 16px and 24px there. But maybe we don't follow that at all? 🤔

Reply:
What I liked about it being smaller is that everything on the page is currently so big compared to the CTA-box, which looks weird IMO.

Reply:
But also, maybe it doesn't need to be a huge headline for the perils section, since it says in the CTA box what insurance we're talking about? It's more for the bundles, to differentiate between the contracts?

Reply:
I think it's good to be clear about that in the bundles as you say. I would vote for larger titles :) But maybe check with Mange and Zak, we will be moving towards having tab navigation and large headlines in the updates to the offer page in the new designs. 
https://www.figma.com/proto/xsUGMTqfaBHgmYE8YX8O2e/Web-Accident-Insurance-PROTOTYPES?page-id=0%3A1&node-id=2%3A893&viewport=388%2C48%2C0.32&scaling=scale-down&starting-point-node-id=2%3A678&show-proto-sidebar=1&hide-ui=1

Reply:
OK changed it back to the larger font